applications/DEMApplication/custom_utilities/post_utilities.h

```diff
@@ -334,76 +320,52 @@ namespace Kratos {
 
         double QuasiStaticAdimensionalNumber(ModelPart& rParticlesModelPart, ModelPart& rContactModelPart, const ProcessInfo& r_process_info) {
 
-            double adimensional_value = 0.0;
-
             ElementsArrayType& pParticleElements = rParticlesModelPart.GetCommunicator().LocalMesh().Elements();
-
-            OpenMPUtils::CreatePartition(ParallelUtilities::GetNumThreads(), pParticleElements.size(), this->GetElementPartition());
-
             array_1d<double,3> particle_forces;
-
             const array_1d<double,3>& gravity = r_process_info[GRAVITY];
-
             double total_force = 0.0;
 
-            //#pragma omp parallel for
             #pragma omp parallel for reduction(+:total_force)
-            for (int k = 0; k < ParallelUtilities::GetNumThreads(); k++) {
-                ElementsArrayType::iterator it_begin = pParticleElements.ptr_begin() + this->GetElementPartition()[k];
-                ElementsArrayType::iterator it_end   = pParticleElements.ptr_begin() + this->GetElementPartition()[k + 1];
+            for (int k = 0; k < (int) pParticleElements.size(); k++) {
```

Comment:
you can also use the parallel utilities here

Reply:
Thanks @philbucher, 
do you mean putting all those into a block_for_each format instead?

Reply:
yes I would recommend that, then it is cleaner (IMO) and future proof

you can also check the Wiki for more details: https://github.com/KratosMultiphysics/Kratos/wiki/Parallel-Utilities